### ISSUE TITLE
DITA OT 3.3.4 - Error Unable to find plugin

### ISSUE THREAD

**Issue by antony123** [dita-ot/dita-ot#3403]
## Expected Behavior
I created a basic plugin by following the instructions given on this page:
https://www.dita-ot.org/dev/topics/html-customization-plugin-bundle-css.html
[com.example.html5-custom-css.zip](https://github.com/dita-ot/dita-ot/files/3801819/com.example.html5-custom-css.zip)

## Actual Behavior
When I run the dita --install command, I get the following error:
**Error: Unable to find plugin com.example.html5-custom-css**

I read the comments related to similar issues. https://github.com/dita-ot/dita-ot/issues/2835
Can anyone tell me what I am doing wrong here.

**Comment by jelovirt** [comment]
@antony123 Did you run

**Comment by antony123** [comment]
> @antony123 Did you run
> 
> ```shell
> dita --install com.example.html5-custom-css
> ```
> 
> Because the [plugin registry](https://www.dita-ot.org/plugins) doesn't have a plugin with that ID.

yes.
This is what I did:

1. I directly created the plugin(s) inside the plugins directory of the DITA OT.
2. Ran `dita --install`. I got the error I mentioned. I tried again `dita --install com.example.html5-custom-css`. No effect. I ran `dita --plugins` to check the names of the installed plugins. My custom ones were not included.

After I saw @raducoravu's comment, I followed his approach approach. I created the plugin outside the DITA OT folder and compressed it. And then I ran `dita --install C:\path\to\Downloads\com.example.html5-custom-css.zip`. It worked. I remember we didn't have to zip the plugin to install it. I could place it within the plugins directory and simply run `dita --install`. Won't that work any more? What was I doing wrong? I followed the instructions on this page: https://www.dita-ot.org/dev/topics/pdf-customization-example.html
